seqMatcher.py

```diff
@@ -2,31 +2,39 @@
-    file1 = toText(tokens1)  #cleaned up code - greatly increases effectiveness of plagiarism checker
-    tokens2 = tokenize(filename2)
-    file2 = toText(tokens2)
-    SM = SequenceMatcher(None, file1, file2)
-    similarity_ratio = SM.ratio()
-    print(similarity_ratio)   # ratio of plagiarised content
-    blocks = list(SM.get_matching_blocks()) #elements  of blocks[] - (start-file1, start-file2, length)
-    blocks = blocks[: -1]
-    f1 = open(filename1, "r")
-    for i in blocks:
-        flag = 0
-        for j in range(len(tokens1)):
-            if tokens1[j][2] == i[0]:  #linking start of matching block to position in cleaned up code
-                start = tokens1[j][1]  #linking position in cleaned up code to position in original code file
-                flag = 1
-            if tokens1[j][2] == (i[0] + i[2] - 1): #linking end to cleaned up code
-                end = tokens1[j][1]  #linking to original code file
-                break
-        if not flag == 0 and (end - start) > 100:  #printing significant blocks of plagiarized content
-            #the start and end of matching blocks is linked to the original code to properly mark the plagiarized content
-            f1.seek(start, 0)
-            print(f1.read(end - start))
+class seqMatcher():
+    def __init__(self,fn1,fn2):
+        self.fn1=fn1
+        self.fn2=fn2
+
+    def plagerised_ratio(self):
+        filename1=self.fn1
+        filename2 = self.fn2
+        tokens1 = tokenize(filename1) #(elements of cleaned up code, their position in original code, position in cleaned up code)
+        file1 = toText(tokens1)  #cleaned up code - greatly increases effectiveness of plagiarism checker
+        tokens2 = tokenize(filename2)
+        file2 = toText(tokens2)
+        SM = SequenceMatcher(None, file1, file2)
+        similarity_ratio = SM.ratio()# ratio of plagiarised content
+        # print(similarity_ratio)   
+        blocks = list(SM.get_matching_blocks()) #elements  of blocks[] - (start-file1, start-file2, length)
+        blocks = blocks[: -1]
+        f1 = open(filename1, "r")
+        copied = ""
+        for i in blocks:
+            flag = 0
+            for j in tokens1:
+                if j[2] == i[0]:  #linking start of matching block to position in cleaned up code
+                    start = j[1]  #linking position in cleaned up code to position in original code file
+                    flag = 1
+                if j[2] == (i[0] + i[2] - 1): #linking end to cleaned up code
+                    end = j[1]  #linking to original code file
+                    break
+            if not flag == 0 and (end - start) > 100:  #printing significant blocks of plagiarized content
+                #the start and end of matching blocks is linked to the original code to properly mark the plagiarized content
+                f1.seek(start, 0)
+                # print(f1.read(end - start))
+                copied+=f1.read(end - start)
```

Comment:
Instead of `copied+=f1.read(end - start)` append the `f1.read(end-start)` to the list and use string join method. Make the copied variable as list type instead of string type.